Source/WebCore/style/values/primitives/StylePrimitiveNumericTypes.h

```diff
@@ -94,7 +94,7 @@ template<CSS::Range nR = CSS::All, CSS::Range pR = nR, typename V = double> stru
     }
 
     constexpr NumberOrPercentageResolvedToNumber(Percentage percentage)
-        : value { percentage.value / 100.0 }
+        : value { percentage.value / static_cast<V>(100.0) }
```

Comment:
I probably would omit the `.0` now that we are explicitly casting to the appropriate numeric type.

Reply:
Good call.